### PR TITLE
initial proposal

### PR DESCRIPTION
**Pull request by goern**
This is a proposal of LABELs supporting Nulecule/developer tooling experiments, read more about it at http://b4mad.net/datenbrei/archives/2015/11/25/making-nuleculization-of-applications-faster/

### CODE REVIEW

**Comment by rhatdan** [comment]
These are fine with me.

File: vendor/projectatomic/labels.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Nulecule Specification Metadata
+
+The Nulecule Specification has been a place to experiment with packaging
+multi container applications. To open the opportunity to generate more of the
+meta information used within a Nulecule file, a set of LABLE to be used on
```
Comment (aweiteka) [inline vendor/projectatomic/labels.md]: LABELs

**Comment by aweiteka** [comment]
LGTM. I can merge now or you can update to address my one comment.

**Comment by goern** [comment]
Thanks guys, and happy St. Patrick's Day!

**Pull request merged**